packages/shared/src/common/date.ts

```diff
@@ -15,3 +15,10 @@ export const optionalDateSchema = z
   .trim()
   .nullish()
   .refine(isValidISODateOptional, { message: "Invalid ISO date" });
+
+export const diffFromNow = (date1?: Date, format: dayjs.UnitTypeLong = "millisecond"): number => {
+  const now = dayjs();
+
+  const diff = dayjs(now).diff(dayjs(date1), format);
```

Comment:
No need to create now and dayjs(now), we're doing the same thing twice there. Just `dayjs().diff(...)` should do.